python_modules/dagster-graphql/dagster_graphql/implementation/loader.py

```diff
@@ -277,3 +283,77 @@ def get_latest_materialization_for_asset_key(self, asset_key: AssetKey) -> Event
     def _fetch(self):
         self._fetched = True
         self._materializations = self._instance.get_latest_materialization_events(self._asset_keys)
+
+
+class BatchAssetDependencyLoader:
```

Comment:
If I understand correctly, this class isn't responsible for loading _all_ asset dependencies, it's only responsible for loading "dependedBy"s that span a repository boundary. It would be helpful to give it a name that communicates that, and for the method names to line up with whatever name that is.

Reply:
Got it--I've adjusted the names of the class and methods:
- `BatchAssetDependencyLoader` -> `CrossRepoAssetDependencyLoader`
- `get_external_asset_deps` -> `get_cross_repo_dependent_assets`

Reply:
In particular, it's just cross-repo _dependedBys_, right? Not all cross-repo dependencies? I think it's worth signalling that in some way. I know I'm being nitpicky on this, but I think these kinds of choices make big differences in how easy it is to evolve this kind of code.